project/maps2params/python/maps_to_params_utils.py

```diff
@@ -256,3 +256,32 @@ def is_symmetric(mat, tol=1e-8):
 
 def is_pos_def(mat):
     return np.all(np.linalg.eigvals(mat) > 0)
+
+def calibrate_alm(alms, exp, freq, **nuis_params):
+    '''
+    Calibrates the alms for each frequency of the LAT by the appropriate calibration factor:
+    i.e.: a_lm^X_nu1  -> c^X_nu1 a_lm^X_nu1, with X=T,E
+    c^X_nu1 = nuis_params["calG_all"]*nuis_params["calX_nu1"]
+    where nuis_params is the dictionary of nuisance parameters and nuis_params["calG_all"] is a calibration factor
+    common to all frequencies.
+    '''
+    cal_alms = alms.copy()
+    cal_alms[0] *= nuis_params["calG_all"]*nuis_params["calT_" + str(freq)]
+    cal_alms[1] *= nuis_params["calG_all"]*nuis_params["calE_" + str(freq)]
+    print('Calibrating the %s alm by calT_%s = %s and calE_%s = %s'%(exp,freq,nuis_params["calT_" + str(freq)],
```

Comment:
I know that @thibautlouis is a big fan of this old-school way to concatenate strings but a more modern way is to use [f-string](https://docs.python.org/3/tutorial/inputoutput.html#fancier-output-formatting) : I think it's easier to write (not need to set the data type) and it's also easier to read since we do not have to go to the end to associate the data with the reserved fields. For instance, you can write your print statement this way 
```python
print(f"Calibrating the {exp} alm by calT_{freq} = {nuis_params[f'calT_{freq}']}...")
```

You can also change the two previous lines by
```python
cal_alms[0] *= nuis_params["calG_all"]*nuis_params[f"calT_{freq}"]
cal_alms[1] *= nuis_params["calG_all"]*nuis_params[f"calE_{freq}"]
```